{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests57_94_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_94_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_94_1.py": 1, "Folder_94": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00023859998327679932, "start": 1700337853.1665576, "stop": 1700337853.1667967, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "07050c03335b47688bd151240fea869a", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_94_1.py::test_matrix_of_several_numbers", "location": ["tests57_94_1.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_94_1.py": 1, "Folder_94": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00028081098571419716, "start": 1700337853.1726892, "stop": 1700337853.1729705, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "07050c03335b47688bd151240fea869a", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_94_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_94_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_94_1.py": 1, "Folder_94": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[92]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[92]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_1/Folder_94/tests57_94_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_matrix[k, l] = matrix[i + k, j + l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_1/Folder_94/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[92]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[92]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[92]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_1/Folder_94/tests57_94_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_matrix[k, l] = matrix[i + k, j + l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_1/Folder_94/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[92]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00035407699760980904, "start": 1700337853.1670673, "stop": 1700337853.167422, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "07050c03335b47688bd151240fea869a", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
